RetailCoder.VBE/Inspections/QuickFixes/AssignedByValParameterQuickFixHelper.cs

```diff
@@ -0,0 +1,195 @@
+﻿using Antlr4.Runtime;
+using Antlr4.Runtime.Tree;
+using Rubberduck.Parsing.Grammar;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.VBEditor;
+using Rubberduck.VBEditor.SafeComWrappers.Abstract;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Rubberduck.Inspections.QuickFixes
+{
+    class AssignedByValParameterQuickFixHelper
+    {
+        private Declaration _target;
+        private QualifiedSelection _selection;
+        private HashSet<string> _namesInUse;
+        private ICodeModule _module;
+        public AssignedByValParameterQuickFixHelper(Declaration target, QualifiedSelection selection)
```

Comment:
The name suggests that this is only usable for `AssignedByValParameterQuickFix`.. is that true? If yes, why?